pkg/k8s/from_yaml.go

```diff
@@ -14,11 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// instead of pkg/maya/maya.go
+// Package k8s provides the necessary codebase for K8s implementation.
+// Setups the different yaml files for deployment
+
```

Comment:
@rbo13 
Can you please remove the comment ( // Package k8s ....) and just keep a blank line there.
That would do fine.
Sorry for bugging you.
(e.g. At line no 15 comments end... line no 16 should be a blank line and `package k8s` should be there on line no 17)

Reply:
Okay @sonasingh46 👍 its fine :) will update my PR